tests/components/homekit/test_homekit.py

```diff
@@ -629,7 +669,7 @@ async def test_homekit_reset_accessories(hass):
         )
         await hass.async_block_till_done()
 
-        assert 2 == hk_driver_config_changed.call_count
+        assert hk_driver_config_changed.call_count == 2
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5745133)